sedai/README.md

```diff
@@ -0,0 +1,76 @@
+
+## **[Schedule a Demo][2]**
+
+## Overview
+Sedai is an autonomous, agentless, and configuration-free SaaS that intelligently manages your cloud environments to prevent availability issues and optimize performance and cost — all in production. Sedai's Continuous Autonomous Availability Management (CAAM) platform helps you take full advantage of your microservices with smart signals that prevent downtime, avoid potential SLO breaches, improve latency, analyze code releases, and lower cloud costs.
+
+**Unlock the full potential of your microservices:**
+* Achieve 99.99% or better application availability
+* Autonomously reduce resource latency by 75%
+* Reduce cloud costs by 30% through autonomous optimization
+* Increase team productivity and agility by 3x
+
+## Core Features
+* **Easy setup:** Our agentless approach integrates with your cloud accounts and observability platforms to get up and running in minutes.
+* **Configuration-free:** We infer your topology and metrics, and intelligently leverage them to detect signals based on observed performance.
+* **Autonomous Availability:** Prevent issues and ensure the highest levels of availability with autonomous actions.
+* **Release Intelligence:** Validate releases' performance in production in real-time with smart scorecards that assess the quality of your deployments and analyze deviations in latency and errors.
+* **Resource Optimization:** Improve customer experience and lower cloud costs on the fly with dynamic allocation that smartly manages your resources based on their traffic needs.
+
+## Home Page
+[Sedai Inc.][1]
+
+## Tecnhical Documentation
+[Sedai User Guide][4]
+
+## Fulfillment Method
+Software as a Service (SaaS)
+
+## Pricing
+### Sedai Autonomous Cloud Application Management
+| Tier 	              | Features               | app instances | Lambda invocations | 1 MONTH | 12 MONTHS |
+| :----	              | :------:               | :-----------: | :----------------: | :-----: | :-------: |
+| Starter	          | Core Features          |       100     |        20M 	    |    $90  |    $900   |
+| Professional Tier 1 | Starter + Remediations |       100     |        20M 	    |   $340  |  $3,420   |
+| Professional Tier 2 | Starter + Remediations |       500     |       100M 	    | $1,300  | $13,000   |
+| Professional Tier 3 | Starter + Remediations |     2,000     |       500M 	    | $4,800  | $48,000   |
+
+### Additional usage fees
+You will be billed monthly for additional usage costs if your usage exceeds your contract. Your additional usage costs will be determined by the number of units you use above your contract.
+
+| Tier       	      | per instance/pod | per 10k invocations |
+| :---       	      | :--------------: | :-----------------: |
+| Professional Tier 1 |      $3.40       |       $0.170        |
+| Professional Tier 2 |      $2.60       |       $0.130        |
+| Professional Tier 3 |      $2.40       |       $0.096        |
```

Comment:
Could this be removed from the readme, and instead have a link to sedai website having the latest information?

Reply:
done